jflex/examples/simple/src/main/jflex/simple.flex

```diff
@@ -1,11 +1,13 @@
-/* this is the scanner example from the JLex website 
+/* this is the scanner example from the JLex website
    (with small modifications to make it more readable) */
 
 %%
 
+%public
```

Comment:
I also don't understand why the class needs to be public.

Reply:
Restriction of the ant run task. Annoying, but not much we can do about it I’d we don’t want to make the example more complicated.